apps/web/src/pages/Home/ExecutiveHomePage/ExecutiveHomePage.tsx

```diff
@@ -0,0 +1,523 @@
+                  id: "ZA20CY",
+                  description: "Text displayed for gaining hiring experience",
+                })}
+              </p>
+            </CardFlat>
+          </div>
+        </div>
+      </div>
+      <SkewedImageContainer
+        imgSrc={executiveProfileHero}
+        imgProps={{
+          "data-h2-background-position":
+            "base(80% 110%) l-tablet(60% 50%) desktop(right 50%)",
+        }}
+      >
+        <p
+          data-h2-font-size="base(h6, 1.4)"
+          data-h2-font-weight="base(300)"
+          data-h2-color="base(white)"
+          data-h2-margin="base(0, 0, x2, 0)"
+          data-h2-max-width="p-tablet(50%)"
+        >
+          {intl.formatMessage({
+            defaultMessage:
+              "Your profile is at the heart of the platform. Tell your story, show how you developed your skills, and use your profile to apply for jobs. Whether you're hunting for a job now or just thinking about the future, a strong profile is your path to new opportunities.",
+            id: "naE1xF",
+            description:
+              "Description of how application profiles works for managers/executives.",
+          })}
+        </p>
+        <div
+          data-h2-display="base(flex)"
+          data-h2-gap="base(x1)"
+          data-h2-justify-content="base(flex-start)"
+        >
+          <Link
+            color="quinary"
+            mode="cta"
+            href={paths.myProfile()}
+            icon={UserPlusIcon}
+          >
+            {intl.formatMessage({
+              defaultMessage: "Create a profile",
+              id: "7hUWc+",
+              description: "Link text for users to create a profile",
+            })}
+          </Link>
+        </div>
+      </SkewedImageContainer>
+      <FlourishContainer show={["bottom"]} skew={false} size="sm">
+        <Heading
+          level="h2"
+          data-h2-margin="base(0, 0, x0.5, 0)"
+          Icon={SparklesIcon}
+          color="primary"
+        >
+          {intl.formatMessage({
+            defaultMessage: "EXposition",
+            id: "7tv7ct",
+            description:
+              "Heading for exposition section on the executive homepage",
+          })}
+        </Heading>
+        <Heading level="h3">
+          {intl.formatMessage({
+            defaultMessage: "Who is EXposition for?",
+            id: "VKE4nf",
+            description: "Heading for exposition eligibility",
+          })}
+        </Heading>
+        <p data-h2-margin="base(x1 0)">
+          {intl.formatMessage({
+            defaultMessage:
+              "Are you a leader looking to attract or promote high-potential talent within the digital (IM, IT or Data fields) community? Do you have talented employees to nominate at the EX minus one to the EX-03 group and level? Are you from the human resources community and interested in interdepartmental opportunities for your digital workforce?",
```

Comment:
So, I'm not sure if this is correct but my assumption is that Data is referring to a field of study so it could make sense as a proper noun "Data field". Though, I am not super certain about that 🤷‍♀️

Reply:
@gobyrne would you mind weighing in on this please?

Reply:
I guess lower case is better?  (doing my part to support Matt's campaign against capital letters)

Reply:
Lets do lowercase @mnigh

Reply:
Thanks everyone 🙇‍♀️ I have all the remaining changes done in [update copy on exec page](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7294/commits/3bc3112fe4da6d1b92217b9830a3909b63d2f453)